package.json

```diff
@@ -104,7 +104,8 @@
     "tailwindcss": "^3.0.23",
     "ts-jest": "^29.1.1",
     "tsup": "^8.0.1",
-    "typescript": "^5.2.2"
+    "typescript": "^5.2.2",
+    "util": "^0.12.5"
```

Comment:
doesn't look like the package-lock was updated after this change

Reply:
done